packages/create-next-app/templates/app-tw/ts/gitignore

```diff
@@ -34,3 +34,4 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+next.compiled.config.*
```

Comment:
@SukkaW 

The reason why I've moved from `.next` to the root dir is because of the path difference.
For example, the path `./something` needs to be modified as `../something`.
Any better idea to resolve this issue?

Reply:
> @SukkaW
> 
> The reason why I've moved from `.next` to the root dir is because of the path difference. For example, the path `./something` needs to be modified as `../something`. Any better idea to resolve this issue?

Yeah that would be a problem.

Maybe tweaking the `jsc.baseUrl` a bit could help?

Reply:
I tried on the swc playground but couldn't find the right config.. I agree that moving to `.next` is the best option though.

Reply:
This is resolved since we use `next.comiled.config.js` on the same level as `next.config.ts`, then unlink it afterwards.

Reply:
This is resolved, the config is stored to the distDir as `next.comiled.config.*js`, and the imported paths are transpiled as relative to the root.